bazel_tools/packaging/package-app.sh

```diff
@@ -116,14 +116,16 @@ elif [[ "$(uname -s)" == "Darwin" ]]; then
   cp $SRC $WORKDIR/$NAME/$NAME
   chmod u+w $WORKDIR/$NAME/$NAME
   function copy_deps() {
-    local from=$1
-    local needed="$(/usr/bin/otool -L "$from" | sed -n -e '1d' -e 's/^\s*\([^ ]*\).*$/\1/p')"
-    loader_path="$ORIGIN"
-    local rpaths="$(/usr/bin/otool -l $from | sed -n '/cmd LC_RPATH/{n;n;p;}' | sed -n -e 's/^.*path \([^ ]*\).*$/\1/p' | sed -e "s|@loader_path|$loader_path|")"
+    local from_original=$(readlink -f $1)
```

Comment:
This was already broken before but we didn’t hit it. `loader_path` needs to be resolved relative to the library not the executable.